exercises/practice/matching-brackets/HELP.md

```diff
@@ -0,0 +1,39 @@
+# Help
```

Comment:
Could you remove this file? It'll be automatically created and downloaded when using the CLI.

Reply:
done!